docs/11.1/api/core.md

```diff
@@ -1415,16 +1415,23 @@ in the right way.
 
 _core.loadData(data, [source])_
 
-Loads new data to Handsontable. Loading new data resets the cell meta.
-Since 8.0.0 loading new data also resets states corresponding to rows and columns
-(for example, row/column sequence, column width, row height, frozen columns etc.).
+The `loadData()` method:
+- Replaces Handsontable's [`data`](@/api/options.md#data)
+- Resets cells' [configuration options](@/guides/getting-started/setting-options.md)
```

Comment:
I need an explanation what does it mean that it resets the configuration options. Maybe this point should be removed?

The below point (about resetting states) is clear to me, but this one is not.

Reply:
@warpech - please have a look now at the updated version: 64e6f14